src/PrivacyPolicy/PrivacyPolicy.less

```diff
@@ -0,0 +1,8 @@
+.privacy-policy {
```

Comment:
for CSS we try to use BEM instead of nesting:
https://gen3.org/resources/developer/style-guide/#css-style-guide

Reply:
recording conversation here for posterity/thoroughness: the privacy policy HTML will be dynamically generated from markdown, so there's not a way to apply BEM in that case